packages/components/src/draggable/stories/index.tsx

```diff
@@ -81,3 +82,69 @@ export const Default: ComponentStory< typeof Draggable > = DefaultTemplate.bind(
 	{}
 );
 Default.args = {};
+
+const AttachElementIdTemplate: ComponentStory< typeof Draggable > = (
```

Comment:
I don't quite think this is worth the code duplication of adding a separate template. For example, you can add a story description in a JSDoc like this, which eliminates the need for a description rendered inside the story itself:

https://github.com/WordPress/gutenberg/blob/85e2d4e3eaf01073d0011bc2247f5b1c2df516ab/packages/components/src/base-control/stories/index.tsx#L52-L59